include/z64.h

```diff
@@ -1488,93 +1525,106 @@ struct ActorContext {
     /* 0x269 */ UNK_TYPE1 pad269[0x1B];
 }; // size = 0x284
 
+typedef struct {
+    /* 0x00 */ u8   seqIndex;
+    /* 0x01 */ u8   nightSeqIndex;
+    /* 0x02 */ u8   unk_02;
+} SoundContext; // size = 0x4
```

Comment:
```suggestion
} SoundContext; // size = 0x3
```
This structure is too large. It doesn't contain any types with alignment requirements so doesn't pad to 0x4.